apps/docs/content/guides/database/extensions/pgaudit.mdx

```diff
@@ -41,68 +46,389 @@ To disable an extension you can call `drop extension`.
+
+This approach allows you to customize the log message and makes it easier to filter for cron events.
+
+```sql
+-- function executed in a cron job
+CREATE OR REPLACE FUNCTION first_cron_job_func()
+RETURNS VOID AS $$
+BEGIN
+    RAISE LOG 'cron job <first-cron-job> started: %', (SELECT NOW());
+    -- cron logic
+END;
+$$ LANGUAGE plpgsql;
+
+-- cron job
+SELECT cron.schedule(
+    'first-cron-job',
+    '* * * * *', -- every minute
+    $$
+      SELECT first_cron_job_func();
+    $$
+);
+```
+
+### Monitoring the `auth.users` Table
+
+In the worst case scenario, where a privileged roles' password is exposed, you can use pgaudit to monitor if the `auth.user` table was targeted. It should be stated that API requests are already monitored in the [API Edge Network](https://supabase.com/dashboard/project/_/logs/edge-logs) and this is more about providing greater clarity about what is happening at the database level.
+
+Logging `auth.user` should be done in Object Mode and requires a custom role:
+
+```sql
+-- create logging role
+CREATE ROLE auth_auditor NOINHERIT;
+
+-- give role permission to observe relevant table events
+GRANT SELECT ON auth.users TO auth_auditor;
+GRANT DELETE ON auth.users TO auth_auditor;
+
+-- assign auth_auditor to pgaudit.role
+ALTER ROLE postgres SET pgaudit.role TO 'auth_auditor';
+
+```
+
+With the above code, any query involving reading or deleting the auth.users email column will be logged.
+
+## Help! Too many logs
+
+pgaudit is infamous for creating log-blizzards. If not configured mindfully, you can accidentally log all database events, including background tasks. This can generate millions of logs in a few hours, making it too overwhelming and chaotic to find insight in the logs.
+
+The first step to solve this problem is to identify which database users pgaudit is observing:
+
+```sql
+-- find all users monitored by pgaudit
+select
+  rolname,
+  rolconfig
+from pg_roles
+where
+  exists (
+    select
+      1
+    from UNNEST(rolconfig) as c
+    where c like '%pgaudit.role%' or c like '%pgaudit.log%'
+  );
+```
+
+To prevent pgaudit from monitoring the problematic roles, you'll want to change their `pgaudit.log` values to `none` and `pgaudit.role` values to `empty quotes ''`
+
+```sql
+  -- Use to disable object level logging
+  ALTER ROLE <role name> SET pgaudit.role TO '';
+
+  -- Use to disable global and user level logging
+  ALTER ROLE <role name> SET pgaudit.log TO 'none';
+```
+
+## FAQ
+
+#### How can I use pgaudit to debug database functions?
+
+Technically yes, but it is not the best approach. It is better to check out our [function debugging guide](https://supabase.com/docs/guides/database/functions#general-logging) instead.
+
+#### How can I download database logs?
+
+In the [Logs Dashboard](https://supabase.com/dashboard/project/_/logs/postgres-logs) you can download logs as CSVs. Advanced log drain support is coming soon for Team and above plans.
```

Comment:
```suggestion
In the [Logs Dashboard](https://supabase.com/dashboard/project/_/logs/postgres-logs) you can download logs as CSVs. 
```
would avoid mentioning future product offerings in case there are adjustments to pricing plans